package.json

```diff
@@ -31,19 +31,25 @@
     "test": "npm run lint && npm run build && npm run testonly",
     "build": "npm run compile-ts && gen-esm-wrapper . ./.esm-wrapper.mjs",
     "prepack": "npm run build",
-    "compile-ts": "tsc -p tsconfig.json"
+    "compile-ts": "tsc -p tsconfig.json && node lib/oidc/static/create-static-pages.js > lib/oidc/static/static-pages.js"
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "lodash.merge": "^4.6.2",
     "system-ca": "^1.0.2"
   },
   "peerDependencies": {
     "mongodb": "^4.2.2 || ^5.0.0",
     "mongodb-log-writer": "^1.1.3"
   },
   "devDependencies": {
+    "@mongodb-js/compass-components": "^1.6.0",
+    "@mongodb-js/oidc-plugin": "file:../oidc-plugin",
```

Comment:
Will publish a first “real” version to npm after https://github.com/mongodb-js/oidc-plugin/pull/5 lands.